type_check_failed,error_msgs
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_table_stats: No attribute 'cursor' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in get_table_name: No attribute 'cursor' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_XPAT: No attribute 'cursor' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_point_tuple: Name 'p1' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in send: Name 'msg' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in _get_random_stats: Name 'y' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in formfield_for_dbfield: No attribute 'name' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: No attribute 'fields' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: No attribute 'fields' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in clean_social_mission: Name 'data' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in __init__: No attribute 'fields' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in follow_button: No attribute 'id' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in starter: Name 'org' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in publictypes: No attribute 'types' on Tuple[Any, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in nextprefix: Name 'used' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 37, in description: No attribute 'append' on Callable[[Any], Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in open_imports: Name 'imp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in make_state_dict: Name 'state_dict' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in make_state_dict: Name 'state_dict' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in become_invisible_surface: Can't assign attribute 'image' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_jsapi_get_jsapi_signature: Name 'signature' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_menu_get_menu_info: Name 'menu_info' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_upload_media: unsupported operand type(s) for item retrieval: 'media_file: io.StringIO' and ''type': str' [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 6, in test_user_get_group_id: Name 'group_id' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_create_group: Name 'group' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_send_text_message: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_shakearound_search_page: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_shakearound_add_material: No attribute 'client' on io.StringIO [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_shakearound_get_shake_info: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_shakearound_get_apply_status: Name 'res' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_wifi_list_shops: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_wifi_get_qrcode_url: Name 'qrcode_url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_scan_get_merchant_info: Name 'res' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_scan_publish_product: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_scan_set_test_whitelist: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_scan_get_product: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in connect: Function socket.send was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 10, in test_delete: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_get_error: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_get_error_unknown: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_get_error_request_invalid: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in stop: Name 'key' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in kill_one: Name 'process' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_action: Name 'redirect_mode' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __repr__: Name '_repr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __converter: Name 'v' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in to_xml: No attribute 'get' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in to_xml: No attribute 'format' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_multi_article_reply: No attribute 'assertTrue' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in draw: No attribute 'dialogue_box' on List[str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in make_credits: Name 'credit' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in transition_out: Name 'credit' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in transition_in: No attribute 'alpha' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _generate: No attribute 'sd' on List[nothing] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _generate: Name 'queryset' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_int: Name 'val' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_sentence: Name 'value' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_short_sentence: No attribute 'sd' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_long_sentence: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _get_http_exception_handler: Name 'handle_http_exception' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in _encode_basestring: Function Pattern.sub was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __get_exception_data: No attribute 'frames' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in add_script: Name 'matches' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in delete_make_new: Name 'conn' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 5, in user_defined_tables_exist: No attribute 'execute' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_colors: unsupported operand type(s) for item assignment: int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in db_connect: Name 'url' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_send_music_message: Name 'result' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_get_mass_message: Name 'result' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_get_menu: Name 'menu' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_short_url: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_get_wechat_ips: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_get_user_info: No attribute 'client' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_get_user_info_batch: No attribute 'assertEqual' on List[Dict[str, str]] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_get_qrcode_url_with_dict_ticket: Name 'url' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_customservice_upload_headimg: No attribute 'assertEqual' on io.StringIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_can_create_multiple_databases: No attribute 'assert_db_history_does_not_contain' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_ref_data_runs_are_not_tracked: No attribute 'assert_db_history_does_not_contain' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_migration_table_is_created_in_every_referenced_database: No attribute 'assert_db_exec_equals' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_untracked_script_can_be_run_many_times: No attribute 'assertEqual' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_tracked_script_is_not_rerun_if_its_content_changed: No attribute 'assertEqual' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_can_run_scripts_in_several_databases: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_can_have_many_scripts: No attribute 'assertEqual' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_script_can_be_composed_of_several_batches: No attribute 'assertEqual' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in test_bad_migration_file_stops_all_subsequent_migrations: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 24, in setUp: No attribute 'setLevel' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in database_exists: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_delete_make_new_deletes_existing_database: No attribute 'assert_recordcount_equals' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_can_check_if_database_contains_user_defined_tables: Name 'conn' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in test_login: No attribute 'client' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_request: No attribute '_client' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in benchmark_test: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in make_state_dict: Name 'state_dict' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in transition_in: No attribute 'blit' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in startup: No attribute 'map_image' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in startup: No attribute 'title_box' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in check_for_blockers: No attribute 'sprites' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in make_image_list: No attribute 'spritesheet' on List[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 35, in main: No attribute 'setup_states' on dict [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in make_event_dict: Name 'event_dict' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in make_state_dict: Name 'state_dict' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in transition_out: No attribute 'alpha' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 44, in make_selection: No attribute 'items' on None [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 14, in confirm_purchase: No attribute 'allow_input' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __repr__: No attribute 'val' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __init__: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in MorrisTraversal: No attribute 'right' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in strStr: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in canJump: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in generatePalindromes: unsupported operand type(s) for +: Tuple[str, ...] and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in groupStrings: Name 'key' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 11, in hashStr: unsupported operand type(s) for item retrieval: 'base: int' and 'i: Any' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in push: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in sortedArrayToBSTRecu: Name 'mid' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in main: Name 'README' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_short_link: No attribute 'GROUP_VIDEO_ID' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_watch_link: No attribute 'video_id_pattern' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in numDecodings: unsupported operand type(s) for item retrieval: 'prev: int' and 'i: Any' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in convertToTitle: unsupported operand type(s) for -: 'result: str' and '1: int' [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in findMin: unsupported operand type(s) for item retrieval: 'low: Union[float, int]' and 'mid: float' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in findKthLargest: Name 'new_pivot_idx' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in longestConsecutiveHelper: No attribute 'val' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in postorderTraversal: No attribute 'right' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in unpaid_members_data: Name 'bc' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_date_between: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_time: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_ipv6: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_choices_key: No attribute 'sd' on List[Tuple[int, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in firstMissingPositive: unsupported operand type(s) for item retrieval: 'i: int' and 'Any' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in gameOfLife: Name 'J' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in main: No attribute 'databases' on List[str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __build_script_list_and_execute: Name 's' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 4, in run_migrations: No attribute '__build_script_list_and_execute' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in _async_process: Name 'handler' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in getReplayGameEventsDebug: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in read_aligned_bytes: Name 'data' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 19, in read_bits: No attribute '_nextbits' on int [attribute-error]
File ""dummpy_input_file"", line 19, in read_bits: Can't assign attribute '_nextbits' on int [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _blob: Name 'length' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _choice: Name 'tag' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in space_binary_string_by_markers: Name 'm' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in union_set: unsupported operand type(s) for item assignment: Type[set] [unsupported-operands]"
False,
False,
True,"File ""dummpy_input_file"", line 5, in isOneEditDistance: No attribute 'isOneEditDistance' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in lengthOfLIS: unsupported operand type(s) for item retrieval: List[int] and List[int] [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in binarySearch: Name 'mid' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in findStrobogrammaticRecu: Name 'num' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in solver: Name 'j' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in remove: Name 'values' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in add: No attribute '_subclass' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in MergeFrom: Name 'message' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _ExtractSymbols: Name 'nested_type' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in findRepeatedDnaSequences: unsupported operand type(s) for -: 'res: List[nothing]' and '9: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in reverseVowels: unsupported operand type(s) for item retrieval: list and list [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in file_lock: Name 'return_value' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Name 'f' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_required: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in is_mutable: Name 'sub_obj' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __iter__: Name 'field' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in isValidBST: No attribute 'val' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in fourSum: Name 'quad' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in group_iban: unsupported operand type(s) for +: str and int [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 4, in group_reference: unsupported operand type(s) for -: int and str [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in fetch_tfd: Name 'all_data' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in fetch_frey: Name 'matfile' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in fetch_mnist: Name 'train_set' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in interpolate_between_points: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in convert_to_one_hot: unsupported operand type(s) for item retrieval: 'is_three_d: bool' and 'n: int' [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 11, in save_checkpoint: Function sys.setrecursionlimit was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 4)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in np_ortho: Name 'g' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in lengthOfLongestSubstringKDistinct: Built-in function ord was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in maximalSquare: unsupported operand type(s) for item retrieval: 'm: int' and 'Any' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in findMedianSortedArrays: No attribute 'getKth' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in getKth: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in merge: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_json_and_reset: Name 'retval' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _choice: No attribute '_json' on dict [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _real32: No attribute '_json' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _real64: Name 'old_json' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_details: No attribute 'status_code' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in create_svg: Name 'total_count' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in dot_draw: Name 'tmp_image' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _AddDescriptors: Name 'field' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in load_xml_config: Name 'display_description' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 57, in load_xml_config: unsupported operand type(s) for +: str and int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in post_events: No attribute 'post_timeout' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _base64_encode: Function bytes.decode was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in _skip_instance: Name 'exists' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in getReplayGameEvents: Name 'generator' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_pid_from_logname_invalid: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in test_read_pid_invalid: Name 'fpath2' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_is_caffe_log_invalid_prefix: Name 'fpath' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_is_caffe_info_log_invalid_fname: Name 'f' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 19, in setup_class: Name 'f' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_get_rel_readme_path_from_log: Name 'p' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 15, in test_find_line: No attribute 'write' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __str__: Name 'base' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_get_index: Name 'r' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in test_post_host_1_guest: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in chkopts: No attribute 'password' on re.Pattern[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in write_conf: Name 'arr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in set_source_file: Can't assign attribute 'paths' on list [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in read_conf: No attribute '_multidefine' on re.Pattern [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in _value_to_lines: No attribute 'iteritems' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in source_file: Name '_afile' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 64, in load_gray: Function posixpath.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in move: Name 'request' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_wrapper_should_delegate_unknown_attribute_access_to_wrapped_driver: Name 'wrapper' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_click_should_raise_if_element_is_not_displayed_for_timeout_seconds: Name 'wrapper' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_waitfor_raise_if_find_element_return_falsy_value: Name 'wrapper' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in raise_NoViableAlt: Name 'line' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in getString: No attribute 'text' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in read: Name 'x' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in __init__: Can't assign attribute 'caseSensitiveLiterals' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in create: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in paddingLength: Name 'pad_len' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_pid_from_str_invalid_alpha: Name 'x' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in fetch_uci_words: Function list.extend was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in reportError: unsupported operand type(s) for >>: 'print: Callable[[], None]' and 'TextIO' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Can't assign attribute '_retTree' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in reportError: unsupported operand type(s) for >>: 'print: Callable[[], None]' and 'TextIO' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'type' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in toString: Name 'tval' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Can't assign attribute 'line' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in toStringTree: No attribute 'getFirstChild' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'hiddenBefore' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in error: unsupported operand type(s) for >>: 'print: Callable[[], None]' and 'TextIO' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_get_class_count_other_default: Name 'counts' is not defined [name-error]"
False,
True,"invalid syntax (<unknown>, line 11)"
False,
True,"invalid syntax (<unknown>, line 13)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_waitfor_wraps_its_return_value_if_given_eager_arument_is_true: Name 'wrapper' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_attr_raise_if_invoked_from_webdriver_wrapped_object: Name 'wrapper' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_silent_property_raise_AttributeError_if_none_bool_given: Name 'wrapper' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in calc_expected_dim: Name 'dim' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in gaussian_error_nll: Name 'nll' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in softmax: Name 'e_X' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 6)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in generate_int: Name 'output' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in run: Can't assign attribute 'stop' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in run: No attribute 'generate_letters_and_numbers' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in generate_word: Name 'words' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in generate_text: Function random.choice was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in generate_last_name: Name 'last_names' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 2, in run: No attribute 'insert' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in generate_url: Name 'domain' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in run: No attribute 'insert' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _colspec: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _serialize_row: Name 'new_row' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in __init__: Can't assign attribute 'end' on int [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_create_estimate_dict: Name 'correct' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in browse: Name 'endpoint' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_add_sync_folder_failed: Name 'client' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_add_sync_folder_force_succeeded: Name 'client' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_remove_sync_folder: Name 'client' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_get_folder_preference: Name 'client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_set_settings: Name 'client' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 20, in chkopts: No attribute 'mode' on re.Pattern [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 30, in chkopts: No attribute 'type' on re.Pattern [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 42, in process: No attribute 'guest' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 26, in process: No attribute 'enable' on None [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 36, in chkopts: No attribute 'name' on re.Pattern[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 37, in chkopts: No attribute 'type' on re.Pattern [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in chkopts: No attribute 'iqn' on re.Pattern[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 25, in process: No attribute 'iqn' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in composed: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __new__: Name 'new_attributes_dict' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_cassette_load: Name 'a_cassette' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_function_decorated_with_use_cassette_can_be_invoked_multiple_times: 'tuple' object is not callable [not-callable]"
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_before_record_response: Name 'cassette' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_nesting_cassette_context_managers: No attribute 'enter_context' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __exit__: Can't assign attribute '__finish' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_decompress_regular: Name 'resp' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_registered_false_matcher: Name 'testfile' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in source_file: Name 'retval' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _GET: No attribute 'machine' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in validates_icon: Name '_' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_view_mail: Name 'mail' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: Can't assign attribute 'fileName' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 8, in __str__: No attribute 'line' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in __init__: Can't assign attribute 'node' on tuple [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __str__: No attribute 'foundChar' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 35, in __init__: Can't assign attribute 'mismatchType' on tuple [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 40, in __str__: No attribute 'append' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in create_database: unsupported operand type(s) for item retrieval: 'db: None' and ''database.url': str' [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 3, in __init__: No attribute '__engine' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _setdaemon: No attribute 'logger' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_default_set_cassette_library_dir: Name 'my_vcr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in load: Name 'new_cassette' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 40, in __print_settings_rec: No attribute '__paper_size_code' on bytes [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_biff_data: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in compose_document: Name 'node' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in compose_scalar_node: Name 'node' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __excel_date_dt: Name 'epoch' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in __init__: Name 'search_key' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 39, in _add_style: No attribute '_xf_val2x' on Tuple[Any, Any, Any, Any, Any, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _all_fonts: No attribute 'get_biff_record' on bytes [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in inner: No attribute 'next' on Generator[bool, Any, None] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in rollback: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in rename: Name 'r' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in maintenance: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in destroy: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in spider_closed: Name 'file_to_save' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in spider_opened: No attribute 'exporter' on BinaryIO [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in start_logger: Name 'logger' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in start_crawler: Name 'spider' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_cls_export_data: Calling TypeGuard function 'inspect.ismethod' with an arbitrary expression not supported yet [not-supported-yet]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_flush_safing: No attribute 'flush' on logging.Logger [attribute-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in emit: No attribute 'threads' on threading.Thread [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in lookup: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in slookup: Name 'res' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in __load_linktype__: Function imp.load_module was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __str__: No attribute 'src_port' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in new_callback: Name 'headers' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 31, in new_fetch_impl: Name 'v' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in setup_xcall: Can't assign attribute '_xcall_supbook_ref' on Tuple[Any, int, int] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 25, in add_sheet_reference: Name 'msg' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in __window1_rec: No attribute '__wnd_hidden' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __update_row_visible_levels: Name 'temp' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __defaultrowheight_rec: No attribute 'row_default_space_below' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __colinfo_rec: Name 'col' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in __window2_rec: No attribute '__show_grid' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 32, in __panes_rec: Name 'result' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in checkbin: Name 'rfn' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_status: No attribute 'lbl2' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in send_sms: No attribute 'sms_message' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_available_packages: Name 'available_modules' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in check_application_requirements: No attribute 'debug' on list [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_multiple: Name 'response' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_multiple_link: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _build_patchers_from_mock_triples: Name 'patcher' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _recursively_apply_get_cassette_subclass: Name 'replacement_obj' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _build_cassette_subclass: Built-in function issubclass was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in backup_plugin_settings: No attribute '_original_settings' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in find: Name 'match' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in scaffold_form: Name 'form' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Can't assign attribute 'app' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 15, in register: Can't assign attribute '_principal' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in emit: Name 'body' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in before_signal: No attribute 'new' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _sentence_parsed_words: No attribute 'get' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in iter_documents: Name 'doc_ids' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in create_fontdict_file: Name 'o' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in output: unsupported operand type(s) for +: str and int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in file_matches: Name 'matches' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 39, in _copy_application_sources: 'fnmatch' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in check_root: 'NoneType' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 9, in cmd_distclean: No attribute 'global_buildozer_dir' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in cmd_help: No attribute 'usage' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in cmd_setdefault: No attribute 'check_build_layout' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _document_xml: Name 'doc_str' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_home: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in __init__: No attribute 'handler' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __call__: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in run: No attribute 'get_logger' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in __call__: Name 'subcommand' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _pop_finish: No attribute 'commit' on Callable[[str], int] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in FitGaussian: No attribute 'x' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _collect_transitions: Built-in function issubclass was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in showRSSDialog: Name 'dlg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_recipe_env: Name 'env' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __getdict: Name 'ret' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in set_filters: Name 'v' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in __init__: Function re.match was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in extract_nodelist_options: No attribute 'strip' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _googleModel: No attribute 'fields' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _patch: No attribute 'patchers' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in teardown: Name 'patcher' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 4)"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in __getitem__: No attribute 'err' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in strip_units: Name 'err' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in DisplayVariables: No attribute 'parnames' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _parse_score: Name 'line' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in verbose: Name 'string' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _download_by_URL: Function TextIO.write was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in _init_from_archive: Name 'f' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 19, in _create_secrets: Name 'secrets_path' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _list_apps: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in _download_app: No attribute 'content' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Can't assign attribute 'sheet_references' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 34, in prec1_expr: No attribute 'rpn' on bytes [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __adjust_bound_col_idx: No attribute '__min_col_idx' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in findcmd: Name 'clist' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in GoldenFileData: Name 'f' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in _do_args: unsupported operand type(s) for item assignment: int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in add_subcommands: Name 'sc' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _add_version: No attribute '_args_map' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _get_acceleration: Function float.__init__ was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in zipup_libs: Name 'output' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in _CreateAclRule: Name 'returned_rule' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in _UpdateAclRule: Name 'returned_rule' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in postOptions: No attribute 'tapLookup' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in parseArgs: unsupported operand type(s) for item retrieval: tuple and str [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _getSuite: unsupported operand type(s) for item retrieval: 'recurse: bool' and ''tests': str' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __init__: No attribute 'SetSize' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_albums: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 50, in main: No attribute '__iter__' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get_json_path: Name 'json_path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in name: No attribute 'value' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in PlugIn: No attribute '_owner' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in do_GET: No attribute 'send_response' on BinaryIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in get: Name 'access_token' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _PromptForSpreadsheet: Name 'id_parts' is not defined [name-error]"
False,
False,
